apps/marketing/content/blog/public-api.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: 'Building the Documenso Public API - The Why and How'
+description: 'This article talks about the need for the public API and the process of building it. It also discusses the requirements we had to meet and the constraints we had to work within.'
+authorName: 'Catalin'
+authorImage: '/blog/blog-author-catalin.webp'
+authorRole: 'Developer'
+date: 2024-03-08
+tags:
+  - Development
+  - API
+---
+
+This article covers the process of building the public API for Documenso. It starts by explaining why the API was needed for a digital document signing company in the first place. Then, it'll dive into the steps we took to build it. Lastly, it'll present the requirements we had to meet and the constraints we had to work within.
+
```

Comment:
Would it be possible to put this in FPV instead of TPV? 👀

Reply:
I don't know what FPV and TPV is. 😅

Reply:
first person and third person are the acronyms, haven't read the feedback